content/updates/importance-of-efficient-notifications-terraform-automation.md

```
@@ -0,0 +1,123 @@
+
+When we encountered alert fatigue and saw teams getting bogged down with too many notifications that might not be relevant to the appropriate engineer, we understood that we needed to make our notifications and alerts both precise and actionable. The approach of broadcasting all failures to an entire global channel was clearly ineffective. The notifications should tie the TF failed apply to the specific team. More specifically, where appropriate, the notifications should be tied to the engineer who made the code change. They are most likely the person best suited for handling the issue and has the most context.
+
+![Pinging ALL vs individual for Notifications](/img/updates/efficient-notifications-terraform-automation/spacelift-notifications-ping.png)
+
+
+## Notifications through Spacelift’s IaC Automation
+
+We frequently use and champion [Spacelift as the TF automation platform](https://spacelift.io/) for managing the IaC of our clients. One of the big features that we love in Spacelift, is their deep integration and belief in [“policy-as-code”](https://docs.spacelift.io/concepts/policy). We use this to express and execute well-defined policies at various decision points in the IaC management workflow using the [OPA Rego policy language](https://www.openpolicyagent.org/docs/latest/policy-language/).
+
+What makes Spacelift’s policy engine shine for notification use cases is its ability to access detailed information about the TF apply that just occurred, including:
+* who triggered it,
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: asterisk

(MD004, ul-style)